### ISSUE TITLE
Safari blocks popup and throws error redirecting back from the popup

### ISSUE THREAD

**Issue by OzzieOrca** [okta/okta-auth-js#914]
When calling `authClient.token.getWithPopup()`, Safari blocks the popup:

<img width="1311" alt="Screen Shot 2021-08-26 at 4 09 33 PM" src="https://user-images.githubusercontent.com/756501/131047827-bc0c4745-c4bd-4efb-b0a6-db06f0c82afd.png">

If you manually show the popup and proceed to sign in, redirecting back to the original page throws an error because `window.opener` is `null`.

The HTML document at https://signon.okta.com/oauth2/v1/authorize/redirect?okta_key=xxx throws
```
TypeError: null is not an object (evaluating 'parentWindow.postMessage')
```
at line 39 (`parentWindow.postMessage(buildMessageData(), 'https\x3A\x2F\x2Fmydomain.com');`)

<img width="1281" alt="Screen Shot 2021-08-26 at 4 14 05 PM" src="https://user-images.githubusercontent.com/756501/131047829-1b4fc8b1-97c3-4b35-9096-e4d4305d92b6.png">

After allowing the popup once, if you retry sign in, Safari no longer blocks the popup and sign in works fine.

Safari version: Version 15.0 (17612.1.26.1.3)

**Comment by OzzieOrca** [comment]
I discovered Firefox 91.0.2 (64-bit) has the same issue. The popup works fine in Chrome 93.0.4577.63 (Official Build) (x86_64).

**Comment by shuowu** [comment]
@OzzieOrca The issue is fixed in [v5.7](https://github.com/okta/okta-auth-js/releases/tag/okta-auth-js-5.7.0)
Closing the issue, feel free to reopen the issue if the problem still exists.
